packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -137,6 +138,7 @@ const Unit = ({
           {showError && <Cross onClick={onClear} className={styles.crossWrapper} />}
         </div>
 
+        {/* TODO: add id to react-select */}
```

Comment:
There is an example on GitHub of react-select-event being used without label here:
- https://github.com/romgain/react-select-event/issues/51

Copying some of the code from that example, we could just wrap the whole Controller + Select in a div with `data-testId`:

```
<div data-testId={`${testId}-select`}>
  <Controller
	control={control}
	name={displayUnitName}
   
	// ...
    
</div>
```

And then in @storybook/testing-library with react-select-event:

```js
import selectEvent from 'react-select-event';

//...

// Changing the unit of the select
await selectEvent.select(
  within(canvas.getByTestId('plant-spacing-select')).
  getByRole('combobox'),
  'm',
);

// Querying the unit of the select
const unit = await within(
  canvas.getByTestId('length-of-bed-select')).
  findByText('m');

expect(unit).toBeInTheDocument();
```

I think this might be the most reliable way to control the Select; I also tried `.getByRole('combobox')`, clicking, and searching for the unit text, but that was extremely flaky for me.

I haven't tried it with the fully updated Unit yet, though! Just on this current branch.